.github/workflows/docs.yml

```diff
@@ -110,6 +110,7 @@ jobs:
         if: steps.cache-examples.outputs.cache-hit != 'true'
         working-directory: modflow6
         run: |
+          sed -i 's/meson==1.3.0/meson>=1.7.1/g' ../modflow6-examples/etc/requirements.pip.txt
```

Comment:
If we want to continue with a floating version in the examples repo, I can unpin 1.3.0 and do `meson!=1.8.0` or maybe `!= 1.8`. maybe this substitution could replace the entire line regardless of what comes after meson?